pkg/cli/cmds/secrets_encrypt.go

```diff
@@ -38,10 +38,10 @@ func NewSecretsEncryptSubcommands(status, enable, disable, prepare, rotate, reen
 			SkipFlagParsing: false,
 			SkipArgReorder:  true,
 			Action:          status,
-			Flags: append(EncryptFlags, &cli.BoolFlag{
-				Name:        "json",
-				Usage:       "Output status as JSON.",
-				Destination: &ServerConfig.EncryptJSON,
+			Flags: append(EncryptFlags, &cli.StringFlag{
+				Name:        "output,o",
+				Usage:       "Output status as (text|json).",
```

Comment:
Maybe use the same-ish text as https://github.com/k3s-io/k3s/pull/5132/files#diff-d5e2e7e02234be3ba6aa1fb1e0727c07cadd6b002383c3a64e67d11f2d9cffd5R133